src/checks/y_check_db_access_in_ut.clas.abap

```diff
@@ -32,7 +32,8 @@ CLASS y_check_db_access_in_ut DEFINITION PUBLIC INHERITING FROM y_check_base CRE
 ENDCLASS.
 
 
-CLASS y_check_db_access_in_ut IMPLEMENTATION.
+
+CLASS Y_CHECK_DB_ACCESS_IN_UT IMPLEMENTATION.
```

Comment:
to lower

Reply:
by default abaplint ignores this, but you can enable it, https://rules.abaplint.org/keyword_case/ try setting `ignoreLowerClassImplmentationStatement` to true